packages/issm/package.py

```diff
@@ -1,62 +1,135 @@
-# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
-# Spack Project Developers. See the top-level COPYRIGHT file for details.
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
 #
 # Copyright 2023 Angus Gibson
 # Modified by Justin Kin Jun Hew, 2025
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack.package import *
+import sys
 
 
 class Issm(AutotoolsPackage):
     """Ice-sheet and Sea-Level System Model"""
 
     homepage = "https://issm.jpl.nasa.gov/"
-    git = "https://github.com/ISSMteam/ISSM.git"
+    git = "https://github.com/ACCESS-NRI/ISSM.git"
 
-    version("develop", branch="main")
-    version("4.24", sha256="0487bd025f37be4a39dfd48b047de6a6423e310dfe5281dbd9a52aa35b26151a")
-    
     maintainers("justinh2002")
 
+    version("upstream", branch="main", git="https://github.com/ISSMteam/ISSM.git")
+    
+    version("main", branch="main", git="https://github.com/ACCESS-NRI/ISSM.git")
+    version("access-development", branch="access-development", git="https://github.com/ACCESS-NRI/ISSM.git")
+
+    variant("wrappers", default=False,
+            description="Enable building with MPI wrappers")
+    #
+    # Build dependencies
+    #
     depends_on("autoconf", type="build")
     depends_on("automake", type="build")
-    depends_on("libtool", type="build")
-    depends_on("m4", type="build")
-
+    depends_on("libtool",  type="build")
+    depends_on("m4",       type="build")
     depends_on("mpi")
     depends_on("petsc+metis+mumps+scalapack")
     depends_on("m1qn3")
 
     conflicts("%gcc@14:", msg="ISSM cannot be built with GCC versions above 13")
+    #
+    # Optional libraries
+    #
+    depends_on('access-triangle', when='+wrappers')
+    depends_on("parmetis",  when="+wrappers")
+    # depends_on("python@3.9.0:3.9", when="+wrappers", type=("build", "run"))
```

Comment:
```
  >> 193    checking for Python header file Python.h... configure: error: Pytho
            n.h not found! Please locate this file and contact ISSM developers 
            via forum or email.
```

Reply:
Looks like you need to re-enable this `depends_on`. Remove the version numbers. i.e. `depends_on("python", when="+wrappers", type=("build", "run"))`